Makefile

```diff
@@ -131,8 +131,17 @@ ifeq ($(TARGET_NAME),TARGET_NANOX)
     SDK_SOURCE_PATH += lib_blewbxx lib_blewbxx_impl
 endif
 
+ifeq ($(TARGET_NAME),TARGET_NANOS2)
+# load fails with --apiLevel 1 on nanosplus
+APP_LOAD_PARAMS_NANO2 = $(subst --apiLevel 1,,$(APP_LOAD_PARAMS))
+endif
```

Comment:
@shufps this one is weird. Is your nanosplus up to date? e.g. as of now running 1.1.0 SE firmware?

Reply:
@xchapron-ledger Thank you! You are right, my "fix" was not working with 1.1 anymore. Removing it, get it to work again :blush: